### DISCUSSION TITLE
Vector3d definition

### DISCUSSION THREAD

**Discussion by Debo-rah** [mtex-toolbox/mtex#1291] [Ask Anything] [answered]
Hi everyone  ,

I have a short question to the vector3d definition. When I use grainboundary.direction or the orientation of a specific plane  in specimen coordinates I get vector3d as an output. However, its not a real vector its one set of x,y,z coordinates. My question is, is the origin of this vector always the x,y coordinate of the pixel from which the vector was deduced? or is there a common origin for the whole EBSD map?

I ask because of the 6 vectors of a plane in calcite, I want to identify those three that point in the direction of the grainboundary. But I dont know how to do this. I then want to do the same for the neighbouring grain to then compare those vectors.

Thanks for helping!

**Comment by kilir** [comment, marked as answer]
Hi,
vector3d gives you a direction from the origin (0,0,0). Plotting the direction at  specific position, e.g. the x,y plane such as in an EBSD map, the position vector is usually considered by e.g. vector3d/quiver3.

I'm not entirely sure how this relates to your question with calcite because if you like to compare two or several directions, any position usually does not matter.

Hope that helps,
Rüdiger

**Reply by Debo-rah** [reply]
Hi Rüdiger, 
thanks for answering. Yes my question was not so easy to understand. I wanted to check where two 10-14 vectors meet at a grain boundary.  So i took one of the 10-14 vectors at both sides of the grain boundary and tried to find out where they meet. However when i try to  plot them they point away from each other and not facing each other, since they are both plottet from the same origin. That was my problem. Anyways, i found another way to get what I want. I calculated a vector from the ebsd point of one side of the grain boundary to the other side of the grain boundary and then calculated the angle between my 10-14 and this vector.  A bit complicated.... I tried to model something, therefore this unusual question. 
Thanks for answering!
